### PR TITLE
Add GET /metrics

### LINKED ISSUE #12: Implement /metrics

### PR DESCRIPTION
**Pull request by madrigal-cqv**
**Description**

- Add GET /props

**Motivation**

- Implement core features

**Testing Done**
```
openapi-generator-cli validate -i ~/.../llamacpp-openapi/models/llamacpp.yml
No validation issues detected.
```

**Backwards Compatibility Criteria (if any)**

N/A

### CODE REVIEW

File: models/llamacpp.yml [hunk 1]
```diff
@@ -109,6 +109,17 @@ paths:
...
+    get:
+      operationId: getMetrics
+      summary: Get the metrics of the server
+      responses:
+        '200':
+          description: The metrics are fetched
+          content:
+            text/plain:
+              schema:
+                type: string
```
Comment (ProfHugo) [inline models/llamacpp.yml:122]: nit: add response for when metrics is disabled on the server

**Review by ProfHugo — DISMISSED** [review]
error mode handling nonblocking for now; can fix later on need-basis

File: models/llamacpp.yml (hunk 1 above)
Comment (madrigal-cqv) [reply models/llamacpp.yml:122]: added

**Review by ProfHugo — APPROVED** [review]
lgtm

**Pull request merged**